docker-compose.yml

```diff
@@ -18,17 +18,19 @@ volumes:
     labels:
       - "nuvlaedge.volume=True"
       - "nuvlaedge.deployment=production"
+      - "nuvlaedge.volume.type=legacy-db"
```

Comment:
I would not add this one because AFAIK the update will fail because it require deletion and recreation to change the labels.
@ignacio-penas can you check ?

Reply:
Hm, I was not aware of that, testing